sccp/data/scRNA_CellCount.csv

```diff
@@ -0,0 +1,47 @@
+Drug,0
+Acetylcysteine,903
```

Comment:
Can't this be calculated from the data itself?

Reply:
I couldn't figure out a way since the tensor is based on the largest number of cells for an experiment.

Reply:
But the filler rows are all zeros, so you can find them that way?

Longer-term, I have a half-finished PR to remove the need for this padding...

Reply:
Used a different method now!